src/app/youtube/components/search/search.component.ts

```diff
@@ -15,63 +17,79 @@ import { YoutubeService } from '../../services/youtube.service';
   providers: [SortByFieldPipe, FilterByWordPipe],
 })
 export class SearchComponent implements OnInit, OnDestroy {
-  public filteredResponse: ISearchResponse = Object.assign({});
+  public filteredCards: ISearchCard[] = [];
   public nothingFound = false;
-  public searchTerm = '';
   public errorMessage = false;
   public isLoading = false;
-  private searchResults: ISearchResponse = Object.assign({});
-  private sub: Subscription = new Subscription();
+  public searchTerm = '';
+  public youtubeError$ = this.store.select(CardSelectors.selectYoutubeError);
+  public customResults$ = this.store.select(CardSelectors.selectCustomCards);
+  private youtubeResults$ = this.store.select(CardSelectors.selectYoutubeCards);
+  private searchCards: ISearchCard[] = [];
```

Comment:
it would be good to use separated service for communication with Store, and here use only function from this service (some kind of Facade)